tutorials/working_with_auxiliary_data.md

```diff
@@ -0,0 +1,89 @@
+# Working with auxiliary data
+
+This tutorial gives an overview of how to deal with auxiliary information in recommender systems using Cornac. In our context, auxiliary data stands for information beyond user-item interactions or preferences, which often hold a clue on how users consume items. Examples of such information or modalities are: item textual descriptions, user/item reviews, product images, social network, etc.
```

Comment:
This tutorial gives an overview of how to `work` with auxiliary information in recommender systems using Cornac. In our context, auxiliary data stands for information beyond user-item interactions or preferences, which often `holds` a clue on how users consume items. Examples of such information or modalities `are` item textual descriptions, user/item reviews, product images, social `networks`, etc.